backend/src/main/resources/config/config.yml

```diff
@@ -26,7 +26,7 @@ projects:
       username: OVERRIDE_IN_LOCAL_CONFIG
       password: OVERRIDE_IN_LOCAL_CONFIG
   - id: nuernberg.sozialpass.app
-    importUrl: https://example.com
+    importUrl: https://data.entitlementcard.app/2023-01-16-nuernberg-akzeptanzstellen.csv
```

Comment:
should be renamed with final version

Reply:
https://data.entitlementcard.app/nuernberg-akzeptanzstellen.csv

Reply:
this is outdated, was already updated